webapp/src/Entity/Language.php

```diff
@@ -298,6 +298,8 @@ public function getAceLanguage(): string
                 return 'prolog';
             case 'rb':
                 return 'ruby';
+            case 'rs':
+                return 'rs';
```

Comment:
```suggestion
                return 'rust';
```

I didn't look into the code that much but should this be rust so https://github.com/ajaxorg/ace-builds/blob/master/src/mode-rust.js is used for the syntax in Ace?

Reply:
i'm not sure sorry

Reply:
should i click "commit suggestion" ?

Reply:
If you submit something in rust and as jury look at the submission and edit it, do you get the correct syntax highlighting? If not try with my suggestion otherwise you can dismiss it.

Reply:
I changed it to 'rust' and syntax highlighting works